persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -299,3 +300,9 @@ UPDATE User SET team = ? WHERE qualified_id = ?;
 selectNameByMessageId:
 SELECT name FROM User
 WHERE qualified_id = (SELECT Message.sender_user_id FROM Message WHERE Message.id = :messageId AND Message.conversation_id = :conversationId);
+
+selectContactsAmount:
+SELECT COUNT() FROM User WHERE User.connection_status = 'ACCEPTED' AND User.qualified_id != :self_user_id;
```

Comment:
I'd suggest an action-oriented name, something like `countContacts`, it's more concise and clear.